src/components/Property/Tenure.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types'
+
+const Tenure = ({ tenure }) => {
+  let tenureToShow = (tenure) => {
+    if (tenure) {
+      return (
+        <ul className="govuk-tag bg-turquoise">
```

Comment:
Hey where is this `bg-turquoise` being defined in css? I think you may have missed it out 😄